src/reference/00-Getting-Started/05-Basic-Def.md

```diff
@@ -110,7 +69,7 @@ rather than complete Scala statements.
 `build.sbt` may also be
 interspersed with `val`s, `lazy val`s, and `def`s. Top-level `object`s and
 `class`es are not allowed in `build.sbt`. Those should go in the `project/`
-directory as full Scala source files.
+directory as Scala source files.
```

Comment:
Also a link to http://www.scala-sbt.org/0.13/docs/Organizing-Build.html here as well.